packages/dds/tree/src/feature-libraries/treeTextCursor.ts

```diff
@@ -25,16 +25,6 @@ import {
  * It's suitable for testing and debugging,
  * though it could also reasonably be used as a fallback for edge cases or for small trees.
  *
- * The serialized format is valid utf-8, and also includes a json compatible intermediate in memory format.
```

Comment:
I think these notes are still relevant to any user of the tree textual format, and they help explain why this library is not outputting strings, and how to get the strings, what the compact rules for them are etc.

Reply:
Actually, I now see you named the other file treetextFormat, not after its Jsonable nature, so this is probably fine as is.